test/660_proxy_ipam_test.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+
+. ./config.sh
+
+UNIVERSE=10.2.2.0/24
+
+start_suite "Ping proxied containers over cross-host weave network (with IPAM)"
+
+weave_on $HOST1 launch -iprange $UNIVERSE
+weave_on $HOST1 launch-proxy
+weave_on $HOST2 launch -iprange $UNIVERSE $HOST1
+weave_on $HOST2 launch-proxy --ipam
+
+proxy start_container $HOST1 -e WEAVE_CIDR= --name=c1
```

Comment:
Shouldn't this be `proxy docker_on ... run -dt` ?  `start_container` runs the weave image.